app/pages/lab/actions/workflow.js

```diff
@@ -0,0 +1,30 @@
+var apiClient = require('panoptes-client/lib/api-client');
+
+var workflows = apiClient.type('workflows');
+
+var workflowActions = {
+  createWorkflow: function(workflowData, sourceWorkflow) {
```

Comment:
And this is subjective, but as much as possible I've tried to avoid UI components directly manipulating resources' `links` properties, since that's really just an implementation detail of how they're serialized. Maybe this signature could be `createWorkflowForProject(projectID, workflowData)`, and the action can wire up the link?